tests/integration/crunchers/test_cruncher_quantile_rolling_windows.py

```diff
@@ -197,15 +201,16 @@ def test_numerical_relationship(self, use_ratio):
         crunched = res(to_find)
 
         # Calculate the same values numerically
-        xs = large_db.filter(variable="Emissions|CO2")["value"].values
-        ys = large_db.filter(variable="Emissions|CH4")["value"].values
+        xs = larger_db.filter(variable="Emissions|CO2")["value"].values
+        ys = larger_db.filter(variable="Emissions|CH4")["value"].values
         if use_ratio:
             ys = ys / xs
+        # Note that the definition of windows differs between the codes
```

Comment:
There's a difference between `silicone.stats.rolling_window_find_quantiles` and the rolling windows quantile cruncher? Is that just a temporary feature?

Reply:
'Feature'... yeah that's probably best considered a bug.

Reply:
has this been updated?